server/utils/featureToggles.ts

```diff
@@ -13,8 +13,7 @@ export const militaryHistoryEnabled = () =>
   config.featureToggles.militaryHistoryEnabledFrom &&
   isAfter(Date.now(), config.featureToggles.militaryHistoryEnabledFrom)
 
-export const editReligionEnabled = () =>
-  config.featureToggles.editReligionEnabledFrom && isAfter(Date.now(), config.featureToggles.editReligionEnabledFrom)
+export const editReligionEnabled = () => true
```

Comment:
Can we just rip this out completely?

Reply:
Ah yeah this was just for local dev because mine was disabled and I was being quick, but yes we can rip both this and the military history code out now, can probably do this as a followup/separate ticket though as this is big enough 😅